Scripts/DCS-BIOS/lib/modules/aircraft_modules/F4U-1D.lua

```diff
@@ -375,6 +375,8 @@ F4U_1D:defineToggleSwitch("RIGHT_CIRCUIT_BREAKER_INSTRUMENT_RELAY", devices.BREA
 -- Navigation Antenna Control
 -- local NAV_ANTENNA = "Navigation Antenna"
 
+F4U_1D:reserveIntValue(2) -- nav antenna control not currently implemented, draw arg 36 is for the adjusting handle
```

Comment:
This would likely be a two-position on/off, with a max value of 1
```suggestion
F4U_1D:reserveIntValue(1) -- nav antenna control not currently implemented, draw arg 36 is for the adjusting handle
```